### PR TITLE
Publish 21.0.0.4-beta + changes in 21.0.0.3

### PR DESCRIPTION
**Pull request by jakub-pomykala**
In 21.0.0.3: Fixing link in MyFaces + added pointer to significant bug fixes section

### CODE REVIEW

**Review by gljrobinson — APPROVED** [review]
LGTM

File: posts/2021-03-22-ldap-kerberos-21004beta.adoc [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+---
+= LDAP connection support for Kerberos authentication in Open Liberty 21.0.0.4-beta
+Austin Bailey <https://github.com/austin0>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+Open Liberty 21.0.0.4-beta contains new authentication methods for LDAP connections and the complete suite of Jakarta EE 9 API candidates.
+
+With the release of Open Liberty 21.0.0.3, MicroProfile 4.0 and associated components are promoted to the general availability release. For more information, see the link:[Open Liberty 21.0.0.3 release blog post].
```
Comment (mbroz2) [inline posts/2021-03-22-ldap-kerberos-21004beta.adoc]: Fix link

File: posts/2021-03-22-ldap-kerberos-21004beta.adoc [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The  includes the following beta features:
```
Comment (mbroz2) [inline posts/2021-03-22-ldap-kerberos-21004beta.adoc]: The what?

File: posts/2021-03-22-ldap-kerberos-21004beta.adoc [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[21.0.0.4-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+As of the 21.0.0.2-beta release, Open Liberty is the first vendor product to be Jakarta EE Web Profile 9.0 compatible. With the recent 21.0.0.3-beta release, Open Liberty is the first vendor product to be added to the Jakarta EE Platform 9.0 compatibility list.
```
Comment (mbroz2) [inline posts/2021-03-22-ldap-kerberos-21004beta.adoc]: Make "Jakarta EE Platform 9.0 compatibility list" a link to https://jakarta.ee/compatibility/#tab-9

File: posts/2021-03-22-ldap-kerberos-21004beta.adoc [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+[source, xml]
+----
+<kerberos keytab= configFile="${server.config.dir}/security/krb5.conf"/>
+
+<ldapRegistry id="LDAP" realm="SampleLdapADRealm" host="ldap_hostname" port="389" ignoreCase="true" baseDN="DC=example,DC=com" bindAuthMechanism="GSSAPI" krb5Principal="user1@EXAMPLE.COM" ldapType="Custom" />
+----
+
+For more information on `LdapRegistry`, see the link:https://openliberty.io/docs/latest/reference/feature/ldapRegistry-3.0.html[LDAP User Registry documentation].
+
+To enable this new beta function in your app, pull the All Beta Features package and add the LDAP Registry 3.0 feature to your `server.xml` file:
```
Comment (mbroz2) [inline posts/2021-03-22-ldap-kerberos-21004beta.adoc]: "LDAP Registry 3.0 feature" should be "LDAP User Registry 3.0 feature"

**Review by mbroz2 — CHANGES_REQUESTED** [review]

File: posts/2021-03-22-ldap-kerberos-21004beta.adoc (hunk 1 above)
Comment (jakub-pomykala) [reply posts/2021-03-22-ldap-kerberos-21004beta.adoc]: Done

File: posts/2021-03-22-ldap-kerberos-21004beta.adoc (hunk 2 above)
Comment (jakub-pomykala) [reply posts/2021-03-22-ldap-kerberos-21004beta.adoc]: Done

File: posts/2021-03-22-ldap-kerberos-21004beta.adoc (hunk 3 above)
Comment (jakub-pomykala) [reply posts/2021-03-22-ldap-kerberos-21004beta.adoc]: Done

File: posts/2021-03-22-ldap-kerberos-21004beta.adoc (hunk 4 above)
Comment (jakub-pomykala) [reply posts/2021-03-22-ldap-kerberos-21004beta.adoc]: Done

**Review by mbroz2 — APPROVED** [review]

**Pull request merged**